content/tutorials/reactor-event-ifttt.textile

```diff
@@ -0,0 +1,196 @@
+
+h2. Setup
+
+We're going to need an Ably API key for use by your browser application. For our purposes, only one API key is required. If you extend the tutorial with your own functionality (you may wish to create a separate API key for the IFTTT if you need it to have a different set of permissions than those assigned for use by the browser app).
+
+First, log into your Ably dashboard, find the app you're going to be using for this tutorial (the "Sandbox" app is probably fine), then click "Manage App".
+
+<a href="/images/tutorials/reactor/reactor-event-ably-apikey-1.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ably-apikey-1.png" style="width: 100%" alt="Manage your Ably app">
+</a>
+
+Select the "*API Keys*" tab, and click "*Create a new API key*":
+
+<a href="/images/tutorials/reactor/reactor-event-ably-apikey-2.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ably-apikey-2.png" style="width: 100%" alt="Create a new API key">
+</a>
+
+# Enter a name for your API key
+# Select only the "publish" and "subscribe" options
+# Click the "Create key" button
+
+Scroll down if necessary, and copy and paste your new API key to a text file for later use in the application.
+
+h2. Creating an Ably Reactor Rule
+
+We'll need to create a Reactor Event Rule in your Ably dashboard. This is what will trigger the IFTTT event.
+
+# Go to your Ably Dashboard
+# Choose the "Reactor" tab
+# Click "New Reactor Rule" button
+# Click the "Choose" button under "Reactor Event" type
+
+h2. Creating an IFTTT Reactor Event
+
+Choose the "*IFTTT*" service integration option
+
+On the resulting form we'll need an IFTTT key to connect Ably to IFTTT with webhooks. 
+
+# Click the "open IFTTT Webhook Settings" button (this will take you to the IFTTT web app)
+# On the resulting page click "Documentation"
+# Copy your key from the top of the documentation page
+# Go back to your Ably Reactor Integration Rule Settings
+# Paste the key into the "IFTTT Webhook key" input.
+# Give the Webhook event a name (eg channel_message_published)
+# Set up a channel filter if necessary
+# Click "*Create*"
+
+You have now set up an IFTTT Reactor event, but we'll also need to create an Applet on IFTTT for it to trigger.
+
+h2. Creating an IFTTT Applet
+
+# Visit "https://ifttt.com/":https://ifttt.com/ (log in if you have not already)
+# Click "*Explore*" in the top right hand corner
+# Scroll until you find the "Create your own" button (or visit "https://ifttt.com/create"https://ifttt.com/create)
+# Search for "webhooks" in the searchbox
+# Click the "*Webhooks*" button
+# Click the "*Receive a web request*" section
+# Add the name for the Webhook event that you created earlier (eg channel_message_published)
+
+Next we will need to choose a service that we want to hook our published message up to. In this case we're going to choose Slack, but you could use any number or services that IFTTT provides.
```

Comment:
Perhaps instead of "but you could use any number or services that IFTTT provides.", something like "but you could use any number or services that IFTTT connects into." as IFTTT are not providing the service.